### DISCUSSION TITLE
Connect to mongoose nestjs

### DISCUSSION THREAD

**Discussion by Miracle30** [Skn0tt/nextjs-nestjs-integration-example#38] [Q&A] [answered]
Hi, i have a question about how to connect to mongoose and write api to this. I do same Nest's doc and it seem like can't connect to mongoose
![Screen Shot 2022-05-12 at 2 53 35 PM](https://user-images.githubusercontent.com/53804232/168020344-35125479-addf-499f-b409-82a774e9e97b.png)
e

**Comment by Miracle30** [comment]
Here is my code to connect to mongoose
![Screen Shot 2022-05-12 at 2 55 48 PM](https://user-images.githubusercontent.com/53804232/168020839-301570ce-9bdd-4b53-8730-be733466f86c.png)

**Comment by Skn0tt** [comment, marked as answer]
I'm pretty sure this is not a problem with the Next.js x Nest.js integration. Have you asked in https://github.com/nestjs/mongoose?
